sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1116,14 +1109,15 @@ static int get_dailink_info(struct device *dev,
 }
 
 static void init_dai_link(struct device *dev, struct snd_soc_dai_link *dai_links,
-			  int be_id, char *name, int playback, int capture,
+			  int id, char *name, int playback, int capture,
```

Comment:
I personally don't like 'id', it could be the dai id, the be id, something else. Using the be_ prefix was intentional to avoid confusions.